main.nf

```diff
@@ -35,8 +35,71 @@ include { POSTPROC_ONLY } from './workflows/postproc_only.nf'
 // modules
 include { BUILD_ALLELETABLE } from './modules/local/build_alleletable.nf'
 
+def helpMessage() {
+  log.info """
+    Usage:
+      The typical command for running the pipeline is as follows:
+      nextflow run main.nf --readDIR data/testdata --target v4
+
+    Mandatory arguments:
+      --readDIR    Path to folder containing fastq files
+      --target     The amplicon panel version that was used. [Options: 4cast, ama1, v1, v2, v3, v4 (default)]
+
+    Optional arguments:
+      --outDIR                  Path to folder to place final output
+      --sequencer               The sequencer used to produce your data. [Options: miseq, nextseq (default)]
+      --QC_only                 Whether to only run QC related workflows or all workflows [Options: True (default), False]
+      --denoised_asvs           Path to denoised ASVs from DADA2. Used to only run the postprocessing workflow TODO: Add this to readme
+      -profile                  Runtime profile [Options: sge,apptainer, docker, conda]
```

Comment:
I wonder if `profile` and `config` should be in there own Nextflow standard parameters section... the separation here is fine with me, but may help new people understand that the '-' is actually correct